### PR TITLE
feat: add full story script

### PR DESCRIPTION
**Pull request by adschwartz**
## Description:
Adding Full Story to our docs page. Full disclosure, I haven't been able to test if this actually works so a follow up PR may be needed

## Is this change user facing?
NO

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -1,184 +1,190 @@
...
-        ],
-        copyright: `Copyright © ${new Date().getFullYear()} Kurtosis Technologies`,
-      },
-      prism: {
-        theme: lightCodeTheme,
-        darkTheme: darkCodeTheme,
-      },
-      algolia: {
-        appId: 'NTSX40VZB8',
+        ({
```
Comment (h4ck3rk3y) [inline docs/docusaurus.config.js]: did we go from tabs to spaces?

File: docs/static/js/load-fullstory.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+window['_fs_host'] = 'fullstory.com';
+window['_fs_script'] = 'edge.fullstory.com/s/fs.js';
+window['_fs_org'] = 'o-1RQ4W2-na1';
```
Comment (h4ck3rk3y) [inline docs/static/js/load-fullstory.js:3]: guessing this can be shared as this is analytics

**Review by h4ck3rk3y — APPROVED** [review]
don't have much context on how full story works! so trusting you on this :-)

File: docs/docusaurus.config.js (hunk 1 above)
Comment (adschwartz) [reply docs/docusaurus.config.js]: oh i'm just using the auto formatter in intellij. Is tabs our coding standard?

File: docs/static/js/load-fullstory.js (hunk 2 above)
Comment (adschwartz) [reply docs/static/js/load-fullstory.js:3]: Yes public id.

File: docs/docusaurus.config.js (hunk 1 above)
Comment (adschwartz) [reply docs/docusaurus.config.js]: I'll fall back to that

**Pull request merged**